Issue 666: Improve User Guide for TUI Commands and Python Equivalents (#668)
pyansys/pyfluent: doc/source/user_guide/solver_settings.rst

Selecting steady or transient
-----------------------------

> The following example shows a comparison between the TUI command and the

Reviewer: ```suggestion
The following example shows a comparison between the TUI commands and the
```

> python code for enabling and disabling the steady and unsteady solution model.

Reviewer 2: ```suggestion
Python code for enabling and disabling the steady and unsteady solution model.
```

> TUI command

Reviewer 2: ```suggestion
**TUI command**
```

.. code:: scheme

    /define/models/steady yes
    /define/models/unsteady_1st_order yes

> Python command

Reviewer 2: ```suggestion
**Python code**
```

> .. code:: python

Reviewer: The python code launches fluent and reads a case, maybe we can leave that out.  Probably it's ok to assume in the examples that the someone has launched a session as described in the "Launching Fluent locally" section:

https://github.com/pyansys/pyfluent/blob/main/doc/source/user_guide/launching_ansys_fluent.rst

> Selecting pressure-based or density-based solver
> ------------------------------------------------

Reviewer 2: ```suggestion
Selecting a pressure-based or density-based solver
--------------------------------------------------
```

> The following example shows a comparison between the TUI command and the

Reviewer 2: ```suggestion
The following examples show comparisons between the TUI commands and the
```

> python code for enabling and disabling the pressure and density based solver

Reviewer 2: ```suggestion
Python code for enabling and disabling the pressure-based and density-based solver
```

Author: @PipKat Is hyphen (-) allowed?

models.

> TUI command

Reviewer 2: ```suggestion
**TUI command**
```

.. code:: scheme

    /define/models/solver/density-based-explicit yes 
    /define/models/solver/density-based-implicit yes
    /define/models/solver/pressure-based yes

> Python command

Reviewer 2: ```suggestion
**Python code**
```

Defining gravitational acceleration
-----------------------------------
The following example shows a comparison between the TUI command and the
python code for settings the gravitational acceleration

> TUI command

Reviewer 2: ```suggestion
**TUI command**
```

.. code:: scheme

    /define/operating-conditions/gravity yes 0 -9.81 0

> 

Reviewer 2: ```suggestion

**Python code**

```
